Tools/environment_install/install-prereqs-ubuntu.sh

```diff
@@ -15,6 +15,12 @@ SITL_PKGS="libtool libxml2-dev libxslt1-dev python-dev python-pip python-setupto
 ASSUME_YES=false
 QUIET=false
 
+IS_WINDOWS_10_WSL=false
```

Comment:
How do you know this is `WINDOWS_10`?  `IS_WSL` I'd believe...

Reply:
WSL only runs on Win10